polyglot-kotlin/README.md

```diff
@@ -1,57 +1,106 @@
 # Polyglot Kotlin  
-Here is the code:
+
+## Overview
+
+- Supports adhoc task execution via Kotlin lambdas and external scripts.
+- Supports the complete Maven model.
+- Preserves well-known Maven idioms with a Kotlin flavor, allowing easy adoption.
+- Supports Xpp3DOM (XML) configuration using idiomatic Kotlin.
+- Provides idiomatic Kotlin extensions that improve readability and reduce lines of code.
+
+## Kotlin POM Example
+
 ```kotlin
-project {
-    name = "Polyglot :: Kotlin"
```

Comment:
So is this longer and explicit (and more consistent setup with the original pom) no longer supported? If so why?

Reply:
There are 3 different ways to supply the project name. That one is still supported. See `polyglot-kotlin/src/test/resources/unit-tests/name` for each supported variation.

Reply:
Variation 1:
```kotlin
project {
    name = "Sample Library"
}
```
Variation 2
```kotlin
project("Sample Library") {
} 
```
Variation 3
```kotlin
project {
    name("Sample Library")
}
```

Reply:
The unit tests provide examples of each variation for each model element.

Reply:
Great.. thanks for confirming.